packages/eslint-plugin/lib/exports.ts

```diff
@@ -1,2 +1,2 @@
 import plugin = require("./index");
-export default plugin;
+export = plugin;
```

Comment:
This is patch I came for: https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/4a8afcde6b0981a5fad2365e50e0ba25ac78e2f8/docs/problems/FalseExportDefault.md